src/main.cpp

```diff
@@ -65,63 +65,64 @@ class SubchainModule : public EditProcessor {
 
 
 void addChainLink(EditProcessChain & procchain, const string & modulename, Setting & moduleconfig) {
-    if(modulename == "character_counts") {
+    switch (modulename) {
```

Comment:
`modulename` is a `string` which cannot be switched.

Internally it is really a char array which doesn't simply match at compile time for building the branch tables.